department-of-veterans-affairs/va.gov-cms
# VACMS 2899 Prevent Duplicate Reference

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2899

## Testing done


## Screenshots


## QA steps

As and editor
1. Login
2. Create new content items for the following and confirm that the VA Benefits Hub section is required and will not allow duplicated references on save:
- [ ] Q&A, FAQ - multiple Q&As, Checklist, Step-by-Step, Media list - Images, Media list - Videos.

## Definition of Done
